ext/autocomplete/theme.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class AutoCompleteTheme extends Themelet {
+	public function build_autocomplete(Page $page) {
+		$base_href = get_base_href();
+		// TODO: AJAX test and fallback.
+
+		$page->add_html_header("<script src='$base_href/ext/autocomplete/lib/jquery-ui.min.js' type='text/javascript'></script>");
+		$page->add_html_header("<script src='$base_href/ext/autocomplete/lib/tag-it.min.js' type='text/javascript'></script>");
+		$page->add_html_header('<link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1/themes/flick/jquery-ui.css">');
+		$page->add_html_header("<link rel='stylesheet' type='text/css' href='$base_href/ext/autocomplete/lib/jquery.tagit.css' />");
+
+		$page->add_html_header("<script>
```

Comment:
Ah yeah it should be. Wrote the ext prior to making that change and this was to avoid it getting grabbed even if it wasn't loaded.